mne/io/kit/tests/test_kit.py

```diff
@@ -138,6 +138,6 @@ def test_ch_loc():
     # test when more than one marker file provided
     mrks = [mrk_path, mrk2_path, mrk3_path]
     read_raw_kit(sqd_path, mrks, elp_path, hsp_path, preload=False)
-
+    assert_dig_allclose(raw_py.info, raw_bin.info)
```

Comment:
the headshape data was intentionally shrunk for file size reasons, that is, we just took the first 500 points for testing. This is in contrast the dev-head transformations are made based on the marker (hpi coils) and the digitized counterparts (elp).